src/GNDStk/Node/src/child.hpp

```diff
@@ -0,0 +1,81 @@
+
+// -----------------------------------------------------------------------------
+// Node::child(string)
+//
+// Searches the Node's children for a child of the given name. If found,
+// returns the child node in question. Else, returns an empty Node.
+// -----------------------------------------------------------------------------
+
+// const
+const Node &child(
```

Comment:
With a small modification I just recently made, and also considering that the code can deal with the "found" flag, and can emit a diagnostic under some circumstances, it's invariably more than a one-liner. Much of the time that I see codes with one-liners that forward parameters to an internal standard data structure, it raises the question of why a user wouldn't prefer instead to access and use the internal data structure directly, especially because they might be familiar with the data structure already. One-liners have their place, to be sure, but sometimes more detail proves to be appropriate, as I believe it does in this case, when adding substantial new value to an existing internal structure.